src/client/dfuse/dfuse_main.c

```diff
@@ -375,7 +424,7 @@ main(int argc, char **argv)
 		rc = dfuse_bg(dfuse_info);
 		if (rc != 0) {
 			printf("Failed to background\n");
-			return 2;
+			exit(2);
```

Comment:
Do we not want this to be a -DER code?

Reply:
Possibly, although it's the exit status of the program and not logged so it probably doesn't make much difference.  The background code is both delicate and relatively small so I don't want to put too much support structure around it for corner-cases.  Failures here should be fairly localised and obvious.

Reply:
Makes sense!